bin/tictactoe

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+Game.new.start
```

Comment:
great job on placing this in a different file, but all user interaction, inputs and outputs should be in a CLI class. So this would look like 
```ruby 
CLI.new.start
``` 
instead